Sources/AppcuesKit/AppcuesKit.docc/URLSchemeConfiguring.md

```diff
@@ -79,6 +79,17 @@ var body: some Scene {
 
 ## Verifying the Custom URL Scheme
 
-Test that the URL scheme handling is set up correctly by navigating to `appcues-APPCUES_APPLICATION_ID://sdk/debugger` in your browser on the device with the app installed.
+The Appcues debugger allows you to easily validate that the Appcues deeplink is properly configured.
+
+1. Trigger the deeplink in your app with a call to ``Appcues/debug()``.
```

Comment:
`Trigger the deeplink` --> `Launch the debugger`

Reply:
Whoops, thanks 🤦‍♂️